docs/unified/clevo/post-install.md

```diff
@@ -58,6 +58,20 @@ install additional Nvidia drivers.
     sudo reboot
     ```
 
+### Suspend fix for SATA disks
+
+Only affects laptops with M.2 SATA disks experiencing sleep issues (the power
+LED not blinking while the laptop is suspended)
+
+Windows and certain Linux distros such as Ubuntu do not enable the necessary
+power saving tweaks to enable sleep mode while a SATA disk is installed.
+
+1. Execute fixup script:
+
+    ```bash
+    curl -sSf https://raw.githubusercontent.com/Dasharo/dasharo-tools/main/clevo/sata-suspend-fixup | sudo sh
```

Comment:
@mkopec @macpijan Should this command work on NV41PZ?

Reply:
it **might not** as there is a known issue with the NV41PZ and SATA disk IIRC. But it is not critical, as NovaCustom is not planning to ship SATA disks in these units at all. Which makes all SATA tests of low priority.

Reply:
NV4x doesn't have SATA at all